modules/tensor_mechanics/doc/content/source/materials/ADLAROMANCEStressUpdateBase.md

```diff
@@ -142,27 +145,30 @@ overfitting is indicated by a stark difference in the regression fit to training
 To validate the obtained regression coefficient values, initial conditions are given to the ROM and
 VPSC, and the resulting simulations are compared.  
 
-### Implementation
+## Writing a LAROMANCE Stress Update Material
 
-While `ADLAROMCreepStressUpdate` contains the necessary algorithms contained to evaluate the ROM,
-the actual data that comprises the ROM is contained in a separate UserObject that inherets from
-`LAROMData`. The data contained in `LAROMData` is the maximum and minimum values used to calibrate
-the ROM, the transform used to prepare the input for the ROM, and the Legendre Polyonimal
-coefficients.
-
-`ADLAROMCreepStressUpdate` must be run in conjunction with an inelastic strain return mapping stress
-calculator such as [ADComputeMultipleInelasticStress](ADComputeMultipleInelasticStress.md).
+While `ADLAROMANCEStressUpdateBase` contains the necessary algorithms contained to evaluate the ROM, he material specific `LAROMANCE` data is contained in inherited classes.
+Within the `tensor_mechanics` module, a test object `SS316HLAROMANCEStressUpdateTest` is included as
+an example of how a ROM can be formulated. Note that `SS316HLAROMANCEStressUpdateTest` is only a
+test object located in `tensor_mechanics/test/src/`, and is not actively updated nor validated, but
+rather included in order to verify the math contained in `ADLAROMANCEStressUpdateBase`, . The
+material specific ROMs provided in specific MOOSE applications should be utilized. which consists of
```

Comment:
change the period near the end of this line to a comma, or remove the period completely